cli/esx/command.go

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
Suggestion to update the headers to the following but can be left to a seperate PR.

THis has been done on several other projects based on discussion with Chip C.

```go
// © Broadcom. All Rights Reserved.
// The term “Broadcom” refers to Broadcom Inc. and/or its subsidiaries.
// SPDX-License-Identifier: Apache-2.0
```

Reply:
Thanks @tenthirtyam

Reply:
If you'd like help making this change, feel free to DM me and I'll help out with that.